src/autoscaler/metricsforwarder/Makefile

```diff
@@ -0,0 +1,73 @@
+PR_NUMBER ?= $(shell gh pr view --json number --jq '.number')
+DEPLOYMENT_NAME ?= autoscaler-$(PR_NUMBER)
+METIRCSFORWARDER_VM := $(shell bosh -d $(DEPLOYMENT_NAME) vms --json | jq '.Tables | .[] | .Rows | .[] | select(.instance|test("metricsforwarder")) | .instance')
+METRICSFORWARDER_APPNAME ?= "metricsforwarder"
+METRICSFORWARDER_HOSTNAME ?= $(METRICSFORWARDER_APPNAME)
+SYSTEM_DOMAIN ?= "autoscaler.app-runtime-interfaces.ci.cloudfoundry.org"
+
+.PHONY: cf-push
+cf-push: fetch-config build-cf set-security-group
+	cf push $(METRICSFORWARDER_APPNAME) -b binary_buildpack -p . --no-start --no-route -c "./metricsforwarder -c metricsforwarder.yml"
+	cf create-route  "$(SYSTEM_DOMAIN)" --hostname "$(METRICSFORWARDER_HOSTNAME)"
+	cf map-route "$(METRICSFORWARDER_APPNAME)" "$(SYSTEM_DOMAIN)" --hostname "$(METRICSFORWARDER_HOSTNAME)"
+	cf start "$(METRICSFORWARDER_APPNAME)"
+	@$(MAKE) stop-metricsforwarder-vm
+
+.PHONY: fetch-config
+fetch-config: start-metricsforwarder-vm
+	# how to define variables in deployment name
+	mkdir -p assets/certs/policy_db assets/certs/storedprocedure_db assets/certs/syslog_client
+
+	$(eval POSTGRES_IP := $(shell bosh -d ${DEPLOYMENT_NAME} vms --json | jq -r '.Tables | .[] | .Rows  | .[] | select(.instance|test("postgres")) | .ips' ))
```

Comment:
This looks like a `make`-calculation in a piece of shell-script for the make-target. Why don't we use shell-script for these variable assignments? It looks simpler to me just having one abstraction-layer here within the target.